index.html

```diff
@@ -8,7 +8,7 @@
 
   <body>
     <h1 id="title">MICROVERSE SURVEY</h1>
-    <p id="description">Thank you for taking the time to help us improve the platform</p>
+    <p id="description">Thank you all for taking the time to help us improve the platform</p>
     <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
 
     <script  src="js/index.js"></script>
```

Comment:
_**Example of an inline comment:**_

You've added a `script` tag in the middle of the body tag. Although you might have your reasons, I think this is not the best place to put this tag. By convention, <script> tags are usually inserted either inside the `head` tag or just before closing the body tag. Kindly fix this :blush: